include/circt/Dialect/Seq/Seq.td

```diff
@@ -52,10 +52,4 @@ def CompRegOp : SeqOp<"compreg",
   let hasCustomAssemblyFormat = 1;
 }
 
-def LowerSeqToSV: Pass<"lower-seq-to-sv", "mlir::ModuleOp"> {
-  let summary = "Lower sequential ops to SV.";
-  let constructor = "circt::seq::createSeqLowerToSVPass()";
-  let dependentDialects = ["circt::sv::SVDialect"];
-}
-
```

Comment:
Seq hasn't followed this convention yet, but it may be worth moving this to a Passes.td file now.  Though I wouldn't consider this blocking.